bridgebuilder-review: ERR_MODULE_NOT_FOUND — dist is missing core/multi-model-pipeline.js

## Summary

The compiled `bridgebuilder-review` skill ships a `dist/main.js` that imports `./core/multi-model-pipeline.js`, but that file is **not present in the shipped `dist/core/`**. Any invocation of the skill fails immediately with `ERR_MODULE_NOT_FOUND` before network/API calls are ever made.

## Repro

```bash
# from a consumer repo with loa as a submodule (loa HEAD = dde7c5a)
.loa/.claude/skills/bridgebuilder-review/resources/entry.sh --pr 1 --dry-run
```

## Observed

```
node:internal/modules/esm/resolve:274
    throw new ERR_MODULE_NOT_FOUND(
          ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '<abs-path>/.loa/.claude/skills/bridgebuilder-review/dist/core/multi-model-pipeline.js'
  imported from <abs-path>/.loa/.claude/skills/bridgebuilder-review/dist/main.js
    at finalizeResolution (node:internal/modules/esm/resolve:274:11)
  code: 'ERR_MODULE_NOT_FOUND'
}
Node.js v24.11.1
```

## Evidence

- `dist/main.js:7` — `import { executeMultiModelReview } from "./core/multi-model-pipeline.js";`
- `ls dist/core/` returns 11 files (`cache`, `context`, `cross-repo-render`, `ecosystem`, `index`, `lore-loader`, `reviewer`, `schemas`, `template`, `truncation`, `types`) — **`multi-model-pipeline` is absent** (no `.js`, no `.d.ts`, no `.js.map`).

The TypeScript source for `multi-model-pipeline` presumably exists (otherwise `main.ts` wouldn't compile); the build/pack step seems to drop it before distribution. Likely culprits: an `exclude` glob in `tsconfig.json` that accidentally matches this file, or a `files` / `prepack` filter in `package.json`.

## Expected

Skill runs end-to-end, printing a JSON summary `{ reviewed, skipped, errors }` per its docstring.

## Environment

- loa HEAD: `dde7c5a` (main, "fix(spiral): REVIEW→IMPL fix-loop + --resume --force")
- Node.js: v24.11.1
- OS: Linux x86_64

## Impact

Skill is completely unusable from a submodule-consumer. Workaround is an internal agent-driven review that produces the same `grimoires/loa/a2a/bridge-reviews/sprint-N-pr-review.{md,json}` artifact shape — but that's not an acceptable long-term substitute for the skill itself.

## Suggested fix

Re-run the build step and verify `dist/core/multi-model-pipeline.js` is emitted, or (if the pipeline module has been renamed / moved) update `main.ts`'s import path accordingly before the next release.

--- a/.github/workflows/bridgebuilder-dist-smoke.yml
+++ b/.github/workflows/bridgebuilder-dist-smoke.yml
@@ -1,0 +1,90 @@
+# cycle-093 Sprint 2 T1.2 / #607 — pack-release smoke test
+#
+# Guards against a regression where dist/core/*.js files go un-tracked
+# (e.g., because a new .gitignore pattern silently excludes them). The
+# #607 root cause was: .gitignore had a bare `dist/` pattern that ignored
+# newer dist modules like multi-model-pipeline.js; older files were tracked
+# only because they predate the rule.
+#
+# This smoke test catches the class by verifying the shipped dist imports
+# cleanly from a fresh checkout — the same failure mode a submodule
+# consumer would see.
+name: bridgebuilder-dist-smoke
+
+on:
+  pull_request:
+    paths:
+      - '.claude/skills/bridgebuilder-review/**'
+      - '.gitignore'
+      - '.github/workflows/bridgebuilder-dist-smoke.yml'
+  push:
+    branches: [main]
+
+permissions:
+  contents: read
+
+jobs:
+  smoke:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout (no submodules — mirror consumer context)
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Set up Node
+        uses: actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020 # v4.4.0
+        with:
+          node-version: '20'
+
+      - name: Verify dist/core/multi-model-pipeline.js is tracked
+        run: |
+          set -euo pipefail
+          dist_file=".claude/skills/bridgebuilder-review/dist/core/multi-model-pipeline.js"
+          if ! git ls-files --error-unmatch "$dist_file" > /dev/null 2>&1; then
+            echo "::error::$dist_file is not tracked by git"
+            echo "::error::This is the #607 regression class. Check .gitignore rules around 'dist/'."
+            exit 1
+          fi
+          echo "✓ $dist_file is tracked"
+
+      - name: Import smoke test (no ERR_MODULE_NOT_FOUND)
+        run: |
+          set -euo pipefail
+          cd .claude/skills/bridgebuilder-review
+          # Syntax check first — fast failure if the file itself is malformed
+          node --check dist/main.js
+          # Import test — verifies ALL transitive imports resolve.
+          # Skill errors after imports are fine here (no API keys in CI);
+          # we only care that ERR_MODULE_NOT_FOUND does NOT fire.
+          node --input-type=module -e "
+            try {
+              await import('./dist/main.js');
+              console.log('import OK');
+            } catch (e) {
+              if (e && e.code === 'ERR_MODULE_NOT_FOUND') {
+                console.error('REGRESSION: ' + e.message);
+                process.exit(1);
+              }
+              // Any other error is acceptable — imports resolved
+              console.log('imports resolved (post-import error tolerated in CI)');
+            }
+          " || {
+            rc=$?
+            if [[ "$rc" -ne 0 ]]; then
+              echo "::error::Import smoke test failed — likely ERR_MODULE_NOT_FOUND regression"
+              exit "$rc"
+            fi
+          }
+
+      - name: Verify dist/core/ file count vs resources/core/ (catch silent drops)
+        run: |
+          set -euo pipefail
+          src_count=$(find .claude/skills/bridgebuilder-review/resources/core -maxdepth 1 -name '*.ts' | wc -l)
+          dist_count=$(find .claude/skills/bridgebuilder-review/dist/core -maxdepth 1 -name '*.js' | wc -l)
+          echo "resources/core: $src_count .ts files"
+          echo "dist/core: $dist_count .js files"
+          if [[ "$dist_count" -lt "$src_count" ]]; then
+            echo "::error::dist/core has fewer .js files ($dist_count) than resources/core has .ts files ($src_count)"
+            echo "::error::Some modules were dropped from the shipped bundle"
+            exit 1
+          fi
+          echo "✓ dist/core/ covers all resources/core/ modules"

